src/core/gps/qgsgpsconnection.h

```diff
@@ -88,6 +88,20 @@ class CORE_EXPORT QgsSatelliteInfo
      * Signal strength (0-99dB), or -1 if not available.
      */
     int signal = -1;
+
+    bool operator==( const QgsSatelliteInfo &other ) const
+    {
+      return id == other.id &&
+             inUse == other.inUse &&
+             elevation == other.elevation &&
```

Comment:
Is a "strict" equal comparison is right here? Elevation and azimuth are double, so is not better to use `qgsDoubleNear` instead?

Reply:
I would assume that if one is a copy of the other they are bitwise equal.
Or is "equality" something different in this context?

Reply:
No, we'd want bitwise equality here.

Reply:
OK for me!